eth/core/host.go

```diff
@@ -148,20 +150,20 @@ type (
 		// HistoricalPlugin implements `libtypes.Preparable`.
 		libtypes.Preparable
```

Comment:
you can now remove this. basically .Prepare should never be called on the historical plugin